test/clj/puppetlabs/trapperkeeper/services/webserver/jetty9_core_test.clj

```diff
@@ -207,9 +205,13 @@
       (update-in [connector-keyword :so-linger-milliseconds] identity)
       (update-in [connector-keyword :idle-timeout-milliseconds] identity)))
 
+
+(def default-http-connector-config
+  {:jmx-enable "false"})
```

Comment:
Good call; I think maybe I just added that in there when I was trying to find tests that weren't cleaning up properly and were leaving behind polluted state for other tests, but then maybe it turned out that it wasn't actually this namespace that was the culprit, so that change isn't necessary.  Will revert.